src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -148,7 +148,7 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
                 (dirty || (cluster && canNextClusterDetails({ cluster })))
               )
             }
-            onNext={handleOnNext(dirty, submitForm, cluster)}
+            onNext={() => void handleOnNext(dirty, submitForm, cluster)}
```

Comment:
Let's revert for now and fix later

Reply:
You are right. Reverting this change. Let's fix separately. 

For future reference:
The issue is in `handleOnNext` returning `() => Promise<void> | void`.
The `onNext` requires its result but `handleOnNext` should be changed to return only `() => void`.